app/models/grda_warehouse/warehouse_reports/report_definition.rb

```diff
@@ -905,6 +905,15 @@ def self.report_list
           health: false,
         }
       end
+      if RailsDrivers.loaded.include?(:user_permission_report)
+        r_list['Operational'] << {
+          url: 'user_permission_report/warehouse_reports/reports',
+          name: 'User Permission Report',
+          description: 'Summary of active users and their functional permissions',
+          limitable: true,
```

Comment:
Probably this should be limitable: false.  I mis-spoke earlier.

Reply:
✅